torch/csrc/distributed/c10d/TraceUtils.h

```diff
@@ -433,10 +434,11 @@ struct NCCLTraceBuffer {
   size_t max_entries_ = 0;
   size_t next_ = 0;
   size_t id_ = 0;
-  std::map<size_t, std::vector<uint64_t>> pg_id_to_ranks_;
+  std::map<std::string, std::vector<uint64_t>> pg_name_to_ranks_;
```

Comment:
technically this would require a major version bump.  i think thats ok to do though.  don't think we have enough customers that care about BC yet.

Reply:
I will do a version bump in the following PR when deprecating pg_id.